src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerStartContainer.groovy

```diff
@@ -15,12 +15,103 @@
  */
 package com.bmuschko.gradle.docker.tasks.container
 
+import org.gradle.api.GradleException
+import org.gradle.api.InvalidUserDataException
+
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
+import org.gradle.util.ConfigureUtil
+
 class DockerStartContainer extends DockerExistingContainer {
+    @Input
+    @Optional
+    LogPoller poller
+    
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Starting container with ID '${getContainerId()}'."
         def containerCommand = dockerClient.startContainerCmd(getContainerId())
         containerCommand.exec()
+
+        if (getPoller()) {
+            poll(dockerClient)
+        }
+    }
+    
+    void poller(Closure closure) {
```

Comment:
Lets instead use the actual parameters to the LogPoller class to make it absolutely obvious what needs to be passed in.